ocn_020_nrt_rw0_nutrient_mole_concentration/contents/src/__init__.py

```diff
@@ -0,0 +1,600 @@
+from __future__ import unicode_literals
+
+import os
+import sys
+import datetime
+import logging
+import subprocess
+import eeUtil
+import requests
+import time
+import urllib
+import urllib.request
+import gdal
+import numpy as np
+from collections import OrderedDict 
+import json 
+import ftplib
+from osgeo import gdal
+import shlex
+import re
+
+DATA_DICT = OrderedDict()
+
+DATA_DICT['daily'] = {
```

Comment:
Would you mind adding some comments here to explain what info is contained in the dictionary?